### ISSUE TITLE
L.Graph on wiki

### ISSUE THREAD

**Issue by rfsaldanha** [humangeo/leaflet-dvf#100]
Can't find L.Graph reference in the wiki.

**Comment by sfairgrieve** [comment]
I've been bad with documentation - my apologies.  I'll try to post some updates soon.  In the meantime, I hope the examples are somewhat helpful.  Also, let me know if you have any questions.

**Comment by shabala** [comment]
Hello @sfairgrieve ,  I am also facing problems without proper documentations.  Could you please help with a generic example for L.Graph / kindly update the documentation.

**Comment by sfairgrieve** [comment]
@shabala absolutely.  If you tell me what doesn't make sense, I'll try to focus on that in the documentation.  Is the problem that the examples are too complex?  If you have specific questions about things, ask those as well.  Just want to make sure any documentation updates address typical problems people are having with L.Graph.

**Comment by ymz1124** [comment]
The 'US Airports and Flight Data' example is hard to understand without document about L.Graph. Besides, after reading deep into the source core, I  have a question that whether processing data in the front end is a proper visualization solution. In that example, flight data and airport data are stored in JavaScript files. However, in some circumstances, the data is not public.

**Comment by sfairgrieve** [comment]
@ymz1124 I'll try to throw together a simpler example and update the docs at some point soon.  With regard to data not being public, there's nothing that prevents you from querying a graph database or web service and dynamically populating `L.Graph` with data from the server.  Storing data in JavaScript files in the examples is simply done for convenience.
